app/medInria/medLogger.cpp

```diff
@@ -0,0 +1,122 @@
+#include "medLogger.h"
+
+#include <dtkCore>
+#include <medApplication.h>
+#include <medPluginManager.h>
+
+medLogger* medLogger::singleton;
+bool medLogger::logAccessFlag = false;
```

Comment:
I remove the ```qThreadStorage``` flag because it has a bug (from QT) that displays an annoying message at the end, and we don't technically need it since all the code will be executed by the same thread now. It is replaced with a simple boolean.